setup/includes/upgrades/mysql/3.0.0-pl.php

```diff
@@ -35,3 +35,4 @@
 include dirname(__DIR__) . '/common/3.0.0-template-preview.php';
 include dirname(__DIR__) . '/common/3.0.0-remove-content-type-field.php';
 include dirname(__DIR__) . '/common/3.0.0-remove-logs-permission.php';
+include dirname(__DIR__) . '/common/3.0.0-update-smtp-system-settings';
```

Comment:
Here, probably, .php needs to be specified.

Reply:
> Maybe the script should be called wider (`3.0.0-update-system-settings-keys.php`, for example) so that in the future it can be used for other system settings?

No. These are specific to a change and are thus atomic to that change.

Reply:
Slight oversight. I'm guessing this was not tested before being submitted. (squints at @Jako)

Reply:
Sorry, but those update scripts are not easy to test.